### ISSUE TITLE
Midway Space Invaders hardware

### ISSUE THREAD

**Issue by BueniaDev** [Buenia0/libblueberrn-compatibility#5]
Though the titular Space Invaders is beginning to boot in the hardware's current implementation, the other games released for that hardware are still unsupported. More work needs to be done before this machine can be promoted to a more functional state.

**Comment by BueniaDev** [comment]
Quick update: The Space Invaders game is now playable. However, a total of 29 (known) titles were released for this hardware, and at least 8 more games need to reach playable status in order for this machine to be promoted to "Loads" status (29 games * 30 percent playability rate = 8.7, or about 9 titles).

**Comment by BueniaDev** [comment]
Upon further research, I just found that in reality, only 5 titles were released for this hardware. The "29 titles" figure included games that were released before the hardware's namesake title.

The 5 titles released for this hardware were:

Space Invaders (fully playable, no sound)
Space Invaders Part II (not tested)
Lunar Rescue (not tested)
Space Attack II (not tested)
Ozma Wars (not tested)

Following the compatibility rules of this database, only one more title needs to reach "playable" status in order for this machine to be promoted to "Loads" status (5 titles in total * 30 percent playability rate = 1.5, or about 2 titles).

Sorry about the earlier confusion!
